contributors/guide/README.md

```diff
@@ -73,12 +73,16 @@ You get the idea - if you ever see something you think should be fixed, you shou
 
 There are multiple repositories within the Kubernetes community and a full list of repositories can be found [here](https://github.com/kubernetes/).
 Each repository in the Kubernetes organization has beginner-friendly issues that provide a good first issue. For example, [kubernetes/kubernetes](https://git.k8s.io/kubernetes) has [help wanted](https://go.k8s.io/help-wanted) and [good first issue](https://github.com/kubernetes/kubernetes/labels/good%20first%20issue) labels for issues that should not need deep knowledge of the system.
-The `good first issue` label indicates that members have committed to providing extra assistance for new contributors.
+The `good first issue` label indicates that members have committed to providing extra assistance for new contributors. Read more [here](/contributors/devel/help-wanted.md).
 <!-- TODO: review removing this note after 3 months or after the 1.12 release -->
 Please note that while several of the repositories in the Kubernetes community have `good first issue` labels already, they are still being applied throughout the community.
 
 Another good strategy is to find a documentation improvement, such as a missing/broken link, which will give you exposure to the code submission/review process without the added complication of technical depth. Please see [Contributing](#contributing) below for the workflow.
 
+#### Issue Assignment in Github
+
+Often, new contributors ask to be assigned an issue they are willing to take on. Unfortunately, due to GitHub limitations we can only assign issues to [org members](#community) or repo collaborators. Instead, a code owner will remove any `help-wanted` or `good-first-issue` flags and verbally state in an issue comment that it is yours to work on.
```

Comment:
If possible can we try and put the agency back on the new contributor taking the initiative rather than on the already overwhelmed approvers.

I think this can be done by having the contributor leave a comment in the form of `#dibs` or some such. We have this concept in service-catalog, and I've seen it elsewhere.

Reply:
Huh, interesting. This is what I've seen:
New Contributor: I'd like to work on this.
SIG member/code owner: All yours.

I'm not sure that I've seen folks start out with a #dibs thing.

How about something like this:

"Instead, please indicate in a comment that it is your intention to work on the issue. We expect you to follow up on the issue within reasonable timeframe (~1 week) so that it is not kept from other contributors."